public/editor-client/src/aiGlobalStyles/index.ts

```diff
@@ -0,0 +1,30 @@
+import { getStyles, getTypography } from "@/api";
+import { Config } from "@/config";
+import { t } from "@/utils/i18n";
+import { AiGlobalStyles, FontStyle, Palette } from "../types/DefaultTemplate";
+
+export const getAIGlobalStyles = (
+  config: Config
+): AiGlobalStyles<Palette, FontStyle> => {
+  return {
+    async regenerateColors(res, rej) {
+      try {
+        const styles = await getStyles(config);
+
+        res(styles.colorPalette);
+      } catch (e) {
+        rej(t("Failed to load meta.json"));
+      }
+    },
+    async regenerateTypography(res, rej) {
+      try {
+        const data = await getTypography(config);
+
+        res(data.fontStyles);
+      } catch (e) {
+        rej(t("Failed to load resolves for selected DefaultTemplate"));
```

Comment:
review reject message

shouldn't be something like: "failed to load typography..." ?